_episodes_rmd/10-supp-addressing-data.Rmd

```diff
@@ -45,13 +45,15 @@ Lets take a look at this data.
 class(dat)
 ```
 
-R has loaded the contents of the .csv file into a variable called `dat` which is a `data frame`.
+R has loaded the contents of the .csv file into a variable called `dat` which is a `data frame`. A data frame is a list of variables of the same number of rows with unique row names, given class "data.frame".
 
-```{r dimDat}
-dim(dat)
```

Comment:
I agree with this removal, because we already explain it in https://github.com/swcarpentry/r-novice-inflammation/blob/df5f83988a0bf12da7cf4713742a4ac63e0686de/_episodes_rmd/01-starting-with-data.Rmd#L296-L300